docs/components/movement-sensor/wheeled-odometry.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Configure a wheeled-odometry movement sensor"
+linkTitle: "wheeled-odometry"
+type: "docs"
+description: "Configure a wheeled-odometry movement sensor."
+images: ["/icons/components/imu.svg"]
+tags: ["movement sensor", "components", "movement sensor"]
+# SMEs: Rand, Martha
+---
+
+Configure a `wheeled-odometry` movement sensor to implement _wheeled odometry_ on your robot.
+
+_Wheeled odometry_ is the estimation of position, orientation, linear velocity, and angular velocity using the dimensions of a base.
```

Comment:
"and calculating how each variable has changed using the movement of the motors through encoders."

Reply:
Think have addressed @randhid 
If not, lmk and will add more closely word-for-word